cpp/src/DataStorm/Node.cpp

```diff
@@ -11,20 +11,98 @@
 using namespace std;
 using namespace DataStorm;
 
+namespace
+{
+
+    Ice::CommunicatorPtr argsToCommunicator(int& argc, const char* argv[], optional<string_view> configFile)
+    {
+        Ice::PropertiesPtr properties = make_shared<Ice::Properties>(vector<string>{"DataStorm"});
+
+        if (configFile)
+        {
+            properties->load(*configFile);
+        }
+
+        Ice::InitializationData initData;
+        initData.properties = properties;
+
+        return Ice::initialize(argc, argv, initData);
```

Comment:
Can't we instead and extra opt-in prefixes to Ice::initialize, and use that?

Reply:
If I understand correctly, you're thinking an extra overload of `args, argv, optInPrefixes`

Reply:
We already have too many initialize.

The opt-in prefixes are primarily for services that we write. It will be extremely rare for a user to opt-in "IceStorm" or "DataStorm". So I'd rather not provide a more overloads to make this more convenient.